### PR TITLE
Move workflow observe under show command as a flag

### PR DESCRIPTION
**Pull request by feedmeapples**
## What was changed

- Removed `workflow observe` command
- added it under `workflow show`  as a `--follow` flag

## Why?

~ same command 

## Checklist

1. Closes 

2. How was this tested:

3. Any docs updates needed?

### CODE REVIEW

**Review by stevekinney — APPROVED** [review]

**Review by bergundy — APPROVED** [review]
Wondering if we want a separate flag to also follow runs in case of continue as new like how the sdk gets a result

File: cli/flags.go [hunk 1]
```diff
@@ -267,6 +268,11 @@ var flagsForShowWorkflow = []cli.Flag{
 		Name:  FlagResetPointsOnly,
 		Usage: "Only show events that are eligible for reset",
 	},
+	&cli.BoolFlag{
```
Comment (bergundy) [inline cli/flags.go:271]: Does this mean that `-f` works too?

Comment (feedmeapples) [reply cli/flags.go:271]: `--follow` would work. `-f` is not yet set as an alias

**Comment by feedmeapples** [comment]
> Wondering if we want a separate flag to also follow runs in case of continue as new like how the sdk gets a result

could you file an issue for this

**Pull request merged**